tests/entry.rs

```diff
@@ -1,7 +1,8 @@
 extern crate tar;
 extern crate tempfile;
 
-use std::fs::{create_dir, File};
+use std::fs::create_dir;
```

Comment:
I don't care too much but this change seem spurious i.e. unnecessary

Reply:
Didn't realize at the time, not intended. Won't fix since it's not that important